src/ElectricPokemon.java

```diff
@@ -0,0 +1,29 @@
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class ElectricPokemon  extends Pokemon{
+
+    String[] attacks = {"thunderPunch" ,"electroBall" ,"thunder" ,"voltTackle"};
+
+    public ElectricPokemon(String name, int level, int hp, String food, String sound) {
+        super("electric",name, level, hp, food, sound);
```

Comment:
de string `"electric"` hier is een "magische string". Hij wordt hier ineens uit het niets getoverd, zonder dat we enige betekenis of context er bij hebben. Je kunt hier dus beter een klasse variabele voor maken `private static final TYPE = "electric"` Dan weten we tenminste dat het om het type van de pokemon gaat. 

Daarnaast is het ook handig om dit soort strings op 1 plek te definieren (boven aan de klas). Als je het nu wilt aanpassen van "electric" naar "volt", moet je namelijk je hele code door om te kijken waar je overal die string hebt geschreven